src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -723,6 +723,15 @@ const getFileOverwriteConfirm = (name: string) => {
 	};
 };
 
+const getMultipleFilesOverwriteConfirm = (overwritesCount: number, detail: string) => {
```

Comment:
It's weird that details is passed into getMultipleFilesOverwriteConfirm and then concatenated, why not pass in the files and build the message properly

Reply:
I'll do the changes 👍